lib/index.js

```diff
@@ -46,9 +46,9 @@ function EventBus () {
   }
 }
 
-export const eventBus = new EventBus()
+exports.eventBus = new EventBus()

```

Comment:
Can you use revealing module pattern instead if we are going to CommonJS anyway? https://weblogs.asp.net/dwahlin/techniques-strategies-and-patterns-for-structuring-javascript-code-revealing-module-pattern